src/codex_plus/request_logger.py

```
@@ -0,0 +1,91 @@
+        try:
+            payload = json.loads(body)
+            logger.info(f"Parsed payload with keys: {list(payload.keys())}")
+        except json.JSONDecodeError as e:
+            logger.warning(f"Invalid JSON in request body: {e}")
+            return
+        except Exception as e:
+            logger.error(f"Unexpected error parsing JSON: {e}")
+            return
+
+        # Create directory with branch name - async
+        log_dir = Path(f"/tmp/codex_plus/{branch}")
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/codex_plus/"

(S108)